src/main.rs

```diff
@@ -0,0 +1,81 @@
+#[macro_use]
+extern crate chan;
+extern crate irc;
+
+use std::io::prelude::*;
+use std::io::BufReader;
+
+use std::default::Default;
+use irc::client::prelude::*;
+
+use std::thread::spawn;
+
+fn main() {
+    let channels = vec![format!("#hashpipe")];
```

Comment:
Cool...I should open a PR and update the example for `irc` since it also has a syntax error =/